middleware.test.ts

```
@@ -0,0 +1,652 @@
+    mockSupabaseClient.auth.getUser.mockResolvedValue({
+      data: { user: null },
+      error: { message: 'Invalid token' },
+    });
+    const { updateSession: freshUpdateSession } = await import('@/utils/supabase/middleware');
+    const result = await freshUpdateSession(mockRequest);
+    expect(mockSupabaseClient.auth.getUser).toHaveBeenCalled();
+    expect(result).toBeInstanceOf(Object);
+  });
+
+  it('should handle missing environment variables', async () => {
+    delete process.env.NEXT_PUBLIC_SUPABASE_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)